src/main/java/com/minecolonies/colony/materials/QuantityNotFound.java

```diff
@@ -7,6 +7,8 @@
  */
 class QuantityNotFound extends RuntimeException
 {
+    //todo: We need to change registry access
+    @SuppressWarnings ("deprecation")
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Suppressing warnings is not allowed [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1309)